elements/rh-alert/rh-alert.css

```diff
@@ -73,6 +75,8 @@ header {
 
 header ::slotted(:is(h1,h2,h3,h4,h5,h6)) {
   margin: 0;
+
+  --rh-font-family-heading: var(--_font-family);
```

Comment:
The XD kit's version is showing that text in the header should be using Red Hat Text and --rh-font-size-body-text-sm. Font weight's correct.

Reply:
Updated the font size for the header, looks like the font-family was already red hat text?